lib/policyEvaluator/evaluator.js

```diff
@@ -118,6 +127,9 @@ evaluators.meetConditions = (requestContext, statementCondition, log) => {
         // Note: this should be the actual operator name, not the bareOperator
         const conditionsWithSameOperator = statementCondition[operator];
         const conditionKeys = Object.keys(conditionsWithSameOperator);
+        if (conditionKeys.some(key => tagConditions.includes(key)) && !requestContext.getNeedTagEval()) {
```

Comment:
I found out there is a second optional parameter to some for `thisArg` so you could do this.
```suggestion
        if (conditionKeys.some(tagConditions.has, tagConditions) && !requestContext.getNeedTagEval()) {
```

If you like it anyway, it's arguably not much if any better.

Reply:
yeah, that's kind of confusing to me readability-wise, so i'll keep the arrow function

Reply:
Haha yea it is a bit of a stretch for sure.